README.md

```diff
@@ -1,21 +1,21 @@
 # argo-nagios-ams-publisher
 
-## Introduction 
+## Description 
 
-Bridge from Nagios to the ARGO Messaging system. The argo-nagios-ams-publisher is responsible for handling the nagios messages and publishes them to specific topic to the AMS System.
+`argo-nagios-ams-publisher` is component acting as bridge from Nagios to ARGO Messaging system. It's integral part of software stack running on ARGO monitoring instance and is responsible for forming and dispatching messages that are results of Nagios tests. It is running as a unix daemon and it consists of two subsystems:
```

Comment:
is **a** component ....
 It's integral part of software stack **is** running on ARGO monitoring instance